pep/translations/en-us.json

```diff
@@ -567,7 +567,9 @@
     "theme": {
         "themes": {
             "default": "Navy (default)",
-            "tomato": "Tomato"
+            "tomato": "Tomato",
+            "dark": "Dark",
+            "highContrast": "High Contrast"
         }
```

Comment:
i know one of the reqs was to show an associated description of the theme (maybe in a tooltip or below the preference select input as help text?), did we want to complete that in this PR?

Reply:
Skipping for now - re @adistasio